dbt_project.yml

```diff
@@ -326,6 +326,8 @@ models:
       +schema: safe
       ethereum:
         +schema: safe_ethereum
+      optimimsm: 
```

Comment:
typo here, which leads to warnings in dbt-test job:
https://github.com/duneanalytics/spellbook/actions/runs/3930368751/jobs/6720310865#step:8:8